java-util/pom.xml

```diff
@@ -85,6 +85,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.yahoo.datasketches</groupId>
```

Comment:
I thought we would like to keep Memory with DataSketches for now. I don't remember us concluding that we need to move it. What do you think  @leventov  @cheddar  @leerho